device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,195 @@
+            the client is running. The information contained in the data area of this option is contained in one or more 
+            opaque fields that identify details of the hardware configuration.
+          $ref: '../../common/dhcpv6options.yaml#/components/schemas/Dhcpv6Options.VendorClass'
+          x-field-uid: 10
+        vendor_info:
+          description: |-
+            This option is used by clients to exchange vendor-specific information with servers.
+          $ref: '../../common/dhcpv6options.yaml#/components/schemas/Dhcpv6Options.VendorInfo'
+          x-field-uid: 11    
+        fqdn:
+          description: |-
+            DHCPv6 server needs to know the FQDN of the client for the addresses for the client's IA_NA bindings in order to 
+            update the IPv6-address-to-FQDN mapping. This option allows the client to convey its FQDN to the server. The Client 
+            FQDN option also contains Flags that DHCPv6 clients and servers use to negotiate who does which update.
+          $ref: '../../common/dhcpv6options.yaml#/components/schemas/Dhcpv6Options.Fqdn'
+          x-field-uid: 12   
+
+    Device.Dhcpv6client.IaType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Identity Association: Each IA has an associated IAID. IA_NA and IA_TA options represent different types of 
+            IPv6 addresses and parameters accepted by DHCPv6 clients each used in different context by an IPv6 node. IA_NA 
+            is the Identity Association for Non-temporary Addresses option. IA_TA is the Identity Association for Temporary 
+            Addresses option. IA_PD and IA_NAPD options represent one or more IPv6 prefix and parameters. IA_PD is the Identity 
+            Association for Prefix Delegation and IA_NAPD s the Identity Association for Temporary Prefix Delegation.
+          type: string
+          default: iana
+          x-field-uid: 1
+          x-enum:
+            iana:
+              x-field-uid: 1
+            iata:
+              x-field-uid: 2
+            iapd:
+              x-field-uid: 3
+            ianapd:
+              x-field-uid: 4
+        iana:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 2 
+        iapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 3
+        ianapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 4
+
+    Device.Dhcpv6client.IaTimeValue:
+      description: >-
+        The container for the suggested times at which the client contacts the server or any available server.
+      type: object
+      properties:
+        ta_1:
+          description: >-
+            The suggested time at which the client contacts the server from which the addresses were obtained to 
```

Comment:
Do you mean "time after allocation of an IPv6 address after which the client contacts the ..."
If comment is valid , it applies to all the below "suggest time at ..." descriptions below.

Reply:
unit of time needs to mentioned. 
Good to mention special behaviour and meaning of 0  and 0xFFFFFF... ( infinity) 
Check RFC default ?